apiserver/facades/controller/firewaller/state.go

```diff
@@ -58,3 +62,22 @@ func (s stateShim) FirewallRule(service state.WellKnownServiceType) (*state.Fire
 	api := state.NewFirewallRules(s.st)
 	return api.Rule(service)
 }
+
+type Subnet interface {
+	ID() string
+	CIDR() string
+}
+
+func (s stateShim) SubnetByID(id string) (Subnet, error) {
+	//sn, err := s.st.SubnetByID(id)
```

Comment:
need to remove this

Reply:
We doing it in this patch?

Reply:
Yes, didn't want to be uploading and changing the commits as you were reviewing.